MIPS/wallet-fee-change

```diff
@@ -0,0 +1,63 @@
+---
+eip: <to be assigned>
+title: <Reduce Mina Wallet Creation Fee>
+description: <The Proposal is to reduce the wallet creation for from 1Mina to .25Mina at the next hard fork on condition that a permanent solution is worked on.>
+author: <Peter Greystoke with help from Trivo <info@minacrypto.com>>
+discussions-to: <https://forums.minaprotocol.com/t/do-you-think-there-should-be-a-fixed-price-to-set-up-a-mina-wallet/4507/32>
+status: Draft
+type: <Standards Track>
+category (*only required for Standards Track): <Core>
+created: <2021-12-20>
+requires (*optional): <EIP number(s)>
+---
+
+MIP Template
+This is the suggested template for new MIPs.
+
+Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `wallet-fee-change.md`.
+
+The title should be 44 characters or less. It should not repeat the MIP number in title, irrespective of the category.
```

Comment:
You can remove this portion of the template.

Reply:
> DONE 

Thanks for the comments, you are right there is no mention of the long term solution. This is a short term, but positive solution to a big issue. I have added a few sentences in the MIP about the method to find the long term solution.